docs/Makefile

```diff
@@ -38,5 +38,8 @@ regenerate_cli_snippets_and_screenshots:
 regenerate_cli_snippets_and_test:
 	cd ../examples/docs_beta_snippets && tox -e docs_snapshot_update && tox -e docs_snapshot_test && echo "\nSnippets regenerated and tested!"
 
+regenerate_cli_snippets_and_test_debug:
+	cd ../examples/docs_beta_snippets && rm -rf /tmp/cli_snippets && mkdir -p /tmp/cli_snippets && (CLI_SNIPPET_WORKING_DIR=/tmp/cli_snippets tox -e docs_snapshot_update && CLI_SNIPPET_WORKING_DIR=/tmp/cli_snippets  tox -e docs_snapshot_test && echo "\nSnippets regenerated and tested!") || (echo "Snippet regeneration failed. Please check the logs for more information. Working directory: /tmp/cli_snippets" && exit 1)
```

Comment:
No need to block on this if you can't figure out how to do it in `make`, but it would be nice if this command was compatible with preset `CLI_SNIPPET_WORKING_DIR` and just defaulted to `/tmp/cli_snippets` if that wasn't set.